### PR TITLE
send a different DM when ban reason contains "ice"

### LINKED ISSUE #75: [FEATURE]  include link to mental health thingy in ice bans
When we ban an ice account (-> ban message contains "ice"), robbb should include links to mental health stuff in the DM!

https://www.nimh.nih.gov/health/find-help/index.shtml for example

### PR DESCRIPTION
**Pull request by Tanish2002**
Closes #75
This is what it looks like for ice
image

This is what it looks like for normal bans
image

### CODE REVIEW

**Comment by Tanish2002** [comment]
some things I want to say before merging this
1. I don't know if having ice-specific code is good. I think it's better to have a environment var which has keeps a list of banned words and compare those words to ban reason
2. Should I also include Reason in the DM because sometimes mods like to send some videos or some sort of message in the ban reason

**Comment by JavaCafe01** [comment]
I also agree that we shouldn't have ice-specific code (check #77)

**Comment by Tanish2002** [comment]
@elkowar Can you please tell what you want :cry:

File: src/commands/ban.rs [hunk 1]
```diff
@@ -51,15 +51,22 @@ async fn do_ban(
...
 
-    let _ = user
-        .dm(&ctx, |m| {
-            m.embed(|e| {
-                e.title(format!("You where banned from {}", guild.name));
-                e.field("Reason", reason, false)
+    if reason.to_string().contains("ice") {
+        let _ = user
+            .dm(&ctx, |m| -> &mut serenity::builder::CreateMessage {
+               m.content("Hey ice, you where banned once again. Instead of wasting your time spamming here, please consider seeking help regarding your mental health.\nhttps://www.nimh.nih.gov/health/find-help/index.shtml")
```
Comment (MCotocel) [inline src/commands/ban.rs:57]: You might want to change 'you where banned' to 'you were banned'

Comment (Tanish2002) [reply src/commands/ban.rs:57]: My bad, I just copy pasted what elkowar gave me :pensive:
I'll fix that later when I'm on my laptop

Comment (elkowar) [reply src/commands/ban.rs:57]: @Tanish2002 you to fix this

Comment (Tanish2002) [reply src/commands/ban.rs]: done UwU

**Comment by 6gk** [comment]
`!ban @user nice try`

**Comment by 6gk** [comment]
Wait, why do I have the contributor badge?  And since when?  I don't appear on the contributors tab either.

**Comment by elkowar** [comment]
@6gk we (lidl) commited the fetcher.sh stuff in a way that kept the commit history, thus probably that makes you contributor (as you "co-authored" that PR)

**Comment by 6gk** [comment]
Interesting.

Was my concern actually valid?  I don't know rust, but it doesn't appear to
check if 'ice' was in the middle of a word.

>

**Comment by legendofmiracles** [comment]
Yes this would also trigger if it contains `nice` or similar words, from my understanding.

**Comment by 6gk** [comment]
fix™

**Comment by legendofmiracles** [comment]
```rust
.split_whitespaces.any(|w| w == "ice")
```

**Comment by Tanish2002** [comment]
> https://github.com/unixporn/trup-rs/pull/78#issuecomment-82236820

Shit! :kekw:

**Pull request merged**